Samples/JavaKitSampleApp/Sources/JavaKitExample/JavaKitExample.swift

```diff
@@ -44,6 +45,10 @@ extension HelloSwift: HelloSwiftNativeMethods {
     self.name = "a 🗑️-collected language"
     _ = self.sayHelloBack(42)
 
+    let predicate: JavaPredicate<JavaInteger> = self.lessThanTen()!
+    let value = predicate.test(JavaInteger(3).as(JavaObject.self))
+    print("Running a JavaPredicate from swift 3 < 10 = \(value)")
```

Comment:
Not a particular compelling example

Reply:
I think this is fine, just at least some soundness check that it works at all 👍 

I'd put it into a test file in this project and we're good them IMHO. We're a bit too lose with not testing behaviors we're implementing here.